src/OrchardCore/OrchardCore.BackgroundTasks/BackgroundTaskService.cs

```diff
@@ -51,7 +51,7 @@ public void Activate()
                 {
                     var timer = _timers[group];
                     var period = _periods[group];
-                    timer.Change(StartNow, period);
+                    timer.Change(TimeSpan.FromSeconds(1), period);
```

Comment:
`StartNow` is not used anymore, change it to `Delay` and assign `FromSeconds(1)`

Reply:
Okay, no problem, i will do